run_dir/static/js/projects.js

```diff
@@ -593,6 +645,16 @@ function sel_from_ps(preset_type, preset, data){
       $('[name="statusOptions"').prop('checked', false)
       updateStatusBar1($('#statusOptNeedReview'), 'defaultClick');
 
+    }
+    if(preset=='Bioinformatics'){
+      $('[name="statusOptions"').prop('checked', false)
+      updateStatusBar1($('#statusOptNeedReview'), 'defaultClick');
```

Comment:
Should it be Need Review status for the Bioinformatics and Order Status presets? I would think Ongoing projects would be better.

Reply:
I actually have no memory of editing this part, I suspect this is a non optimal diff output. I'll check.